src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -0,0 +1,257 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a Person's birthday in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidBirthday(String)}
+ */
+public class Birthday {
+
+    public static final String MESSAGE_BIRTHDAY_CONSTRAINTS = "Birthday must be in the following format: "
+            + "dd/mm/yyyy, dd.mm.yyyy or dd-mm-yyyy.\nLeading zeroes are allowed for the day and month field. "
+            + "The year field must have 4 digits.\nExample: 21/10/1995, 21-05-1996. 8.10.1987";
+    private static final int[] MONTH_TO_DAY_MAPPING = {31, 28, 31, 30, 31, 30, 31, 31,
```

Comment:
Putting the sentence after `\n` to a new line would be better?

Reply:
Yes will change that